### PR TITLE
Add lazygit to list of guis

### PR DESCRIPTION
**Pull request by jesseduffield**
Lemme know if any changes are required :)

### CODE REVIEW

File: resources/guis.yml [hunk 1]
```diff
@@ -274,37 +274,39 @@ guis:
...
+- name: LazyGit
+  url: https://github.com/jesseduffield/lazygit
+  image_tag: guis/lazygit@2x.png
+  platforms:
+  - Windows
+  - Mac
+  - Linux
+  price: Free
+  license: MIT
+  order: 31
```
Comment (pedrorijo91) [inline resources/guis.yml:286]: there seems to be a problem with the image: https://git-scm-pr-1496.herokuapp.com/downloads/guis

Comment (pedrorijo91) [reply resources/guis.yml:286]: we are missing the normal version besides the 2x. See this PR as an example https://github.com/git/git-scm.com/pull/1485/files

**Review by pedrorijo91 — CHANGES_REQUESTED** [review]
Fix broken image please

File: resources/guis.yml [hunk 2]
```diff
@@ -256,12 +256,12 @@ guis:
   price: Free
   license: MIT
   order: 28
-- name: GitAtomic
-  url: https://gitatomic.bitbucket.io/
-  image_tag: guis/gitatomic@2x.png
+- name: Cong
```
Comment (pedrorijo91) [inline resources/guis.yml:259]: Just for curiosity, why/how did you change the order of these clients? :)

**Comment by pedrorijo91** [comment]
thanks for adding this new client @jesseduffield ! :)

File: resources/guis.yml (hunk 2 above)
Comment (jesseduffield) [reply resources/guis.yml:259]: I followed the instructions I followed the 'sort the tools' instructions in the readme here https://github.com/git/git-scm.com#adding-new-gui

**Comment by jesseduffield** [comment]
Should be good now @pedrorijo91 :)

**Review by pedrorijo91 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**